cmd/osmosisd/cmd/init.go

```diff
@@ -107,7 +107,8 @@ func InitCmd(mbm module.BasicManager, defaultNodeHome string) *cobra.Command {
 			config.P2P.MaxNumOutboundPeers = 60
 			config.Mempool.Size = 10000
 			config.StateSync.TrustPeriod = 112 * time.Hour
-			config.FastSync.Version = "v0"
+			// UNFORKINGTODO OQ: Uncomment this line if we enable fastSync, do we need this?
+			//config.FastSync.Version = "v0"
```

Comment:
I think so? idrk tbh

seems like a decision orthogonal to this PR

Reply:
The problem is the new cometBFT's config no longer has a FastSync field, so this was erroring, so I commented it out with this note.

Reply:
https://github.com/osmosis-labs/osmosis/pull/6413/commits/6e948da474dd8515f96a00188be0fd5e3a8fee35